src/bip44.h

```diff
@@ -64,17 +62,41 @@ bool bip44_isValidStakingKeyPath(const bip44_path_t* pathSpec);
 
 bool bip44_containsMoreThanAddress(const bip44_path_t* pathSpec);
 
-#ifdef POOL_OPERATOR_APP
 bool bip44_isValidPoolColdKeyPath(const bip44_path_t* pathSpec);
 bool bip44_hasReasonablePoolColdKeyIndex(const bip44_path_t* pathSpec);
-#endif // POOL_OPERATOR_APP
 
 bool isHardened(uint32_t value);
 uint32_t unharden(uint32_t value);
 
 size_t bip44_printToStr(const bip44_path_t*, char* out, size_t outSize);
 
 
+typedef enum {
+	// hd wallet account
+	PATH_WALLET_ACCOUNT,
+
+	// hd wallet address
+	PATH_WALLET_ADDRESS,
```

Comment:
PATH_WALLET_SPENDING_KEY would be less ambiguous of a name, as technically, the staking key is also included in addresses